service/create/launch_configuration.go

```diff
@@ -91,21 +93,23 @@ func (s *Service) createLaunchConfiguration(input launchConfigurationInput) (boo
 		return false, microerror.MaskAny(err)
 	}
 
-	launchConfigName, err := launchConfigurationName(input.cluster, input.prefix)
+	launchConfigName, err := launchConfigurationName(input.cluster, input.prefix, securityGroupID)
```

Comment:
Why do we need to add securityGroupID to the name? 

It's a later change but when we support multiple ASGs with different instance types they will need separate launch configs. But I think they would use the same security group.